templates/components/common/navigation-menu.html

```diff
@@ -2,231 +2,96 @@
-                            </a>
-                        </li>
-                        <li>
-                            <a 
-                                class="js-nav-pages-action"
-                                href="{{urls.account.details}}"
-                                aria-label="{{lang 'account.nav.settings'}}"
-                            >
-                                {{lang 'account.nav.settings'}}
-                            </a>
-                        </li>
-                    </ul>
-                </div>
-            </li>
-            <li>
-                <a 
-                    class="js-nav-pages-action"
-                    href="{{urls.auth.logout}}"
-                    aria-label="{{lang 'common.logout'}}"
-                >
-                    {{lang 'common.logout'}}
-                </a>
-            </li>
-        {{else}}
-            <li>
-                <a 
-                    class="js-nav-pages-action"
-                    href="{{urls.auth.login}}"
-                    aria-label="{{lang 'common.login'}}"
-                >
-                    {{lang 'common.login'}}
-                </a>
-                {{#if settings.account_creation_enabled}}
-                    {{lang 'common.or'}}
+                <li>
+                    <a 
+                        class="js-nav-pages-action c-navigation__sign-in"
+                        href="{{url}}"
+                    >
+                        <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none">
+                            <path d="M20 21V19C20 17.9391 19.5786 16.9217 18.8284 16.1716C18.0783 15.4214 17.0609 15 16 15H8C6.93913 15 5.92172 15.4214 5.17157 16.1716C4.42143 16.9217 4 17.9391 4 19V21M12 11C14.2091 11 16 9.20914 16 7C16 4.79086 14.2091 3 12 3C9.79086 3 8 4.79086 8 7C8 9.20914 9.79086 11 12 11Z" stroke="#5A64C1" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
+                        </svg>
+                        {{lang 'common.sign_in_now'}}
+                    </a>
+                </li>
+                <li>
                     <a 
-                        class="js-nav-pages-action"
-                        href="{{urls.auth.create_account}}"
-                        aria-label="{{lang 'common.sign_up'}}"
+                        class="js-nav-pages-action c-navigation__link c-navigation-list__submenu-header"
+                        href="#"
                     >
-                        {{lang 'common.sign_up'}}
+                        {{lang 'common.reach_out'}}
                     </a>
-                {{/if}}
+                    {{> components/common/social-links }}
+                </li>
+            </ul>
+            <li>
+                <a href="/">
+                    <img class="c-navigation__promo" src={{cdn 'img/navigation/30-off.png'}} height="190" width="284" loading="lazy"/>
```

Comment:
It would be good to either move it to a separate component `navigation-promo` or update classname to `navigation-menu__promo` to clearly state where it belongs to. 

Now, based on the classanme I'd search for it in `navigation` component